conduct/code_of_conduct.md

```diff
@@ -39,16 +39,17 @@ Reporting Guidelines (for [online](reporting_online.md) and
 This code applies equally to founders, developers, mentors and new community
 members, in all spaces managed by Project Jupyter (including IPython). This
 includes the mailing lists, our GitHub organizations, our chat rooms, in-person
-events, and any other forums created by the project team. In addition,
+events, community forum, and any other forums created by the project team. In addition,
```

Comment:
Worth being explicit about it being "the Discourse community forum"? Otherwise the language "c. forum and other forums" reads a bit funny. We only have Discourse and so I don't see any danger in being explicit about it. Very minor nit.

Reply:
done - added in a link to the forum as well